instance.tf

```diff
@@ -55,7 +55,7 @@ resource "aws_security_group" "allow_ingress" {
     from_port   = 80
     to_port     = 80
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0"]
+    cidr_blocks = [var.my_ip_address]
```

Comment:
This change is not related to the remote state feature, please remove it from this pull request